sympy/printing/fcode.py

```diff
@@ -570,6 +604,16 @@ def fcode(expr, assign_to=None, **settings):
     >>> fcode(floor(x) + ceiling(x), user_functions=custom_functions)
     '      CEIL(x) + FLOOR1(x)'
 
+    Returned Fortan code can have different variables in comparison to ``expr``
+    whenever there are similar variables with different case.Name mangling
+    can be stopped by setting ``name_mangling`` to ``False``.
+
+    >>> Z, z = symbols('Z,z')
+    >>> fcode(Z - z)
+    '      Z - z_'
+    >>> fcode(Z - z, name_mangling= False)
```

Comment:
nit picking (not important): `name_mangling=False`

Reply:
my output is inconsistent for `python2` and `python3`, you have any clue why?
python3 : 
```
Python 3.5.2 (default, Nov 23 2017, 16:37:01) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from sympy.printing.tests.test_fcode import *
>>> Z, z = symbols('Z,z')
>>> fcode(Z - z)
'      Z - z_'
```
python2:
```
Python 2.7.12 (default, Nov 20 2017, 18:23:56) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from sympy.printing.tests.test_fcode import *
>>> Z, z = symbols('Z,z')
>>> fcode(Z - z)
'      Z - z'
```